packages/core/src/middleware/middleware-service.ts

```diff
@@ -177,6 +178,14 @@ interface IMiddleware {
    * @returns The configuration options for Helmet middleware.
    */
   addHelmet(options?: OptionsHelmet): void;
+
+  /**
+   * Adds Multer middleware for handling multipart/form-data, typically used for file uploads.
+   *
+   * @param options - Optional configuration options for Multer.
+   */
+  addMulter(options?: Multer.MulterOptions): void;
```

Comment:
Multer has a return type multer.Multer. Multer it is an intermediate configuration to be used in the `this.middleware.addMiddleware()`

```typescript
const upload = multer({ dest: "uploads/" });
 this.middleware.addMiddleware("/uploads", upload.single("file"));
```
Also, rather than call addMulter maybe rename to `setupMulter()` as setup multer returns multer.Multer that can be used like demonstrated above. The final and correct use would be:

```typescript
 const upload = this.middleware.setupMulter({ dest: "uploads/" });
 this.middleware.addMiddleware("/uploads", upload.single("file"));
```